### DISCUSSION TITLE
problema para execultar o Beeware

### DISCUSSION THREAD

**Discussion by Mayck20** [beeware/briefcase#1728] [Q&A]
olá estou tento os seguintes problemas ao tentar usar o beeware no Vs code e no pycharm:
primeiro erro --> Briefcase_Home

com não consegui resolver esse problema no meu pc instalei  no notebook e o outro problema ocorreu:
segundo problema --> briefcase run android roda até certo ponto, quando quero criar um emulador ou rodar no meu celular, infelizmente apresenta erro de aceleração de hardware. resolvi isso.. coloquei pra rodar novamente e só ficar rodando e não aparece nada. quando fecho o terminal aparece a tela. poderia me ajudar por favor com alguma dica pra resolver esses problema, já tentei de tudo que estava ao menu alcance. mais não tem muita informação na internet sobre o assunto de forma mais prática e clara.

**Comment by freakboy3742** [comment]
Machine translation:

Olá!

Com base na sua descrição, parece que você está enfrentando https://github.com/beeware/briefcase/issues/1573. Acreditamos que corrigimos isso na versão de desenvolvimento do Briefcase, mas a correção ainda não foi publicada em um lançamento.

No entanto, é muito difícil fornecer assistência de depuração quando tudo o que temos é uma vaga descrição do que você fez. Se você puder fornecer o arquivo de log gerado pelo Briefcase, detalhes do código que você está executando e detalhes do seu próprio computador (sistema operacional, versão do python, etc.), poderemos fornecer mais assistência.

---

Hi!

Based on your description, it sounds like you might be experiencing https://github.com/beeware/briefcase/issues/1573. We believe we have fixed this in the development version of Briefcase, but the fix hasn't been published in a release yet.

However, it's very difficult to provide debugging assistance when all we have is a vague description of what you have done. If you are able to provide the log file that Briefcase generates, details of what code you are running, and details of your own computer (Operating system, python version etc), we may be able to provide more assistance.

**Reply by Mayck20** [reply]
eu estou usando o passo a passo da documentação, não cheguei a desenvolver nada próprio, só segui a documentação e os código que estão fornecidos la. 
minha versão do python é a mais recente 3.12
sistema operacional Windows 10 tanto no notebook quanto no pc
quando sigo o passo a passo da documentação ele roda mais não me permite emular para android e nem para o celular, mais o briefcase dev roda normalmente. 
não sei se a falta da minha placa de video faz alguma diferença nesse desenvolvimento. pq meu pc não tem placa de video.

mais o passo que sigo no pc, tanto no pycharm quanto no vscode:
md beeware-tutorial
cd beeware-tutorial
py -m venv beeware-venv
beeware-venv/Scripts/activate
pip install briefcase
assim que instala eu tento rodar o briefcase new

![erro](https://github.com/beeware/briefcase/assets/154385206/464e4974-860c-4e12-a92a-8065f19b797c)

**Reply by freakboy3742** [reply]
Machine translation:

O erro que você está recebendo sugere que você definiu uma variável de ambiente `BRIEFCASE_HOME` e ela está definida para um local que não existe.

Não sei se esta variável foi definida pelo PyCharm ou por você mesmo - mas de qualquer forma; ele precisa ser excluído ou definido com um valor válido.

---

The error you're receiving suggests that you have defined a `BRIEFCASE_HOME` environment variable, and it is set to a location that doesn't exist.

I don't know if it this variable has been set by PyCharm, or by yourself - but either way; it either needs to be deleted, or set to a valid value.

**Reply by Mayck20** [reply]
meu amigo consegui resolver o erro que você me auxilio. muito obrigado.
eu to iniciando nessa area faz pouco tempo, gostaria de ver só mais uma ajuda sua se possivel for. pq ao rodar o briefcase run android e tentar emular me aparece outro erro que não consigo identificar uma forma de resolver, até instalei o pacote android para tentar mais não foi possivel. olha a mensagem ai.
![erro](https://github.com/beeware/briefcase/assets/154385206/4c251ca0-e0d2-4771-8443-76abe7c8c467)

**Reply by freakboy3742** [reply]
Machine translation:

Infelizmente, não posso dar mais conselhos do que a mensagem de erro já fornece. O problema é que o emulador não está iniciando; isso pode ocorrer porque seu computador não possui os recursos de hardware necessários para executar o emulador ou porque você precisa fazer uma alteração na configuração para permitir a execução do emulador. A documentação vinculada fornece o guia do Android para iniciar um emulador, e o comando fornecido informa como iniciar o emulador manualmente, o que pode exibir erros mais úteis.

---

Unfortunately, I'm not able to give any more advice than the error message is already providing. The issue is that the emulator isn't starting; this may be because your computer doesn't have the hardware features that are needed to run the emulator, or that you need to make a configuration change to allow the emulator to run. The documentation that is linked provides the Android guide for starting an emulator, and the command provided tells you how to start the emulator manually, which may display more helpful errors.

**Comment by Mayck20** [comment]
Valeu amigo.. obrigado pela ajuda

Em qui, 11 de abr de 2024 23:06, Russell Keith-Magee <
***@***.***> escreveu:

**Comment by Mayck20** [comment]
Valeu amigo obrigado pela atenção e ajuda..

Em qui, 11 de abr de 2024 23:08, M&F Maximo ***@***.***>
escreveu:
